src/main.js

```diff
@@ -140,8 +153,9 @@ const getBestCompareUnit = (start, end) => {
   return 'day';
 };
 
-// :: (String) -> Option(Date | Int | String) = new Date() -> Object
-const getAutoCompareRangeAndLabel = (period, baseDate = new Date()) => {
+const getAutoCompareRangeAndLabel = (
+  period: Period, baseDate: Date = new Date(),
```

Comment:
Add `?` to `baseDate` as it is optional?

And should `baseDate` be `DateLike` instead?